WordPress/wordpress-importer
# Test plugin for WordPress 6.3

In this PR, we test the plugin for WordPress 6.3.

## How to test

1. Clone this PR
2. Using `wp-env` run an instance. The minimum PHP version is 7.0 because WordPress 6.3 [dropped support for PHP 5](https://make.wordpress.org/core/2023/07/05/dropping-support-for-php-5/).
3. Install the WordPress Importer plugin
4. `composer install`
5. Go to `http://localhost:8888/wp-admin/admin.php?import=wordpress` and perform an import.
6. Check if the content has been imported and no errors are being generated.
